### PR TITLE
Fix #81: add a search box to the sessions and speaker pages

### LINKED ISSUE #81: Quickly find a session/speaker within the lists on the relevant pages
At the top of the sessions and speakers page, add a jquery filter/search to find a particular session/speaker.

### PR DESCRIPTION
**Pull request by adrianbanks**
Redo of #99. This is rebased onto the top of the current production branch.

### CODE REVIEW

File: DDDEastAnglia/DDDEastAnglia.csproj [hunk 1]
```diff
@@ -308,6 +308,25 @@
...
+      <DependentUpon>T4MVC.tt</DependentUpon>
+    </Compile>
+    <Compile Include="SpeakerController.generated.cs">
+      <DependentUpon>T4MVC.tt</DependentUpon>
+    </Compile>
+    <Compile Include="T4MVC.cs">
+      <AutoGen>True</AutoGen>
+      <DesignTime>True</DesignTime>
+      <DependentUpon>T4MVC.tt</DependentUpon>
+    </Compile>
```
Comment (adrianbanks) [inline DDDEastAnglia/DDDEastAnglia.csproj]: Fixing this now...

File: DDDEastAnglia/DDDEastAnglia.csproj [hunk 2]
```diff
@@ -308,6 +308,8 @@
     <Compile Include="Models\VoteModel.cs" />
     <Compile Include="Mvc\Attributes\AllowCrossSiteJsonAttribute.cs" />
     <Compile Include="Models\VotingCookie.cs" />
+    <Content Include="Scripts\tweetbutton.js" />
```
Comment (adrianbanks) [inline DDDEastAnglia/DDDEastAnglia.csproj]: I haven't added this file. Why is it a diff on my branch?

**Comment by alastairs** [comment]
tweetbutton.js needs to be deleted too - that was removed when I re-did the Twitter button a few days back.

File: DDDEastAnglia/DDDEastAnglia.csproj (hunk 2 above)
Comment (alastairs) [inline DDDEastAnglia/DDDEastAnglia.csproj]: Same problem as the T4MVC files. The line just needs deleting.

**Comment by adrianbanks** [comment]
Ok, good to go.

**Comment by alastairs** [comment]
Ok, just awaiting confirmation from TeamCity.

**Pull request merged**